ci/build.py

```diff
@@ -180,63 +245,133 @@ def default_ccache_dir() -> str:
         return ccache_dir
     return os.path.join(tempfile.gettempdir(), "ci_ccache")
 
+def trim_container_id(cid):
+    return cid[:12]
 
 def container_run(platform: str,
                   docker_binary: str,
                   docker_registry: str,
                   shared_memory_size: str,
-                  local_ccache_dir: str,
                   command: List[str],
+                  local_ccache_dir: str,
+                  cleanup: Cleanup,
                   dry_run: bool = False,
-                  interactive: bool = False) -> str:
+                  interactive: bool = False) -> int:
+    #
+    # Environment setup
+    #
+    environment = {
+        'CCACHE_MAXSIZE': '20G',
```

Comment:
You've changed the default value of this which was 500G previously. This would delete almost the whole cache on the ci. It makes sense to check whether the script is run on the ci and if not, set a lower value. But I would leave it for a later change and just fix the value by using the variable `CCACHE_MAXSIZE`  which was previously there.

Reply:
done